### PR TITLE
[Guides]Code Review -  Edited grammar, added punctuation.

### PR DESCRIPTION
**Pull request by binarygalwalkin**
- [ X] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [ X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X ] My pull request targets the `master` branch of freeCodeCamp.
- [X ] None of my changes are plagiarized from another source without proper attribution.
- [ X] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXX

### CODE REVIEW

File: guide/english/computer-science/data-mining/index.md [hunk 1]
```diff
@@ -1,13 +1,116 @@
 ---
-title: Data Mining
+title: Computor Science > Data Mining
+author: binarygalwalkin
+description: The definition of, and links to resources on Data Mining.
+fcc.custom: Free Code Camp Guides
```
Comment (raisedadead) [inline guide/english/computer-science/data-mining/index.md]: Remove all these custom keys from your change. Please adhere to the schema.

File: guide/english/computer-science/data-mining/index.md [hunk 2]
```diff
@@ -1,13 +1,116 @@
...
+fcc.author: Amber P.
```
Comment (raisedadead) [inline guide/english/computer-science/data-mining/index.md]: Remove all these custom keys from your change. Please adhere to the schema.

File: guide/english/computer-science/data-mining/index.md [hunk 3]
```diff
@@ -1,13 +1,116 @@
...
+fcc.date: 10/25/18
```
Comment (raisedadead) [inline guide/english/computer-science/data-mining/index.md]: Remove all these custom keys from your change. Please adhere to the schema.

**Review by raisedadead — CHANGES_REQUESTED** [review]

**Comment by raisedadead** [comment]
Hi @binarygalwalkin 

Thanks for the PR. Can you please remove these custom keys?

Thanks.

**Comment by binarygalwalkin** [comment]
@raisedadead done!

**Pull request closed without merging**